### DISCUSSION TITLE
Steps to setup it at windows enviroment

### DISCUSSION THREAD

**Discussion by Harsh-kanors** [tchiotludo/akhq#949] [Q&A] [answered]
I m new to java. Please list out all steps to install this at windows 10.

**Comment by tchiotludo** [comment]
What is your need ? develop akhq or simply use it ?

**Reply by Harsh-kanors** [reply]
Simply use it...

**Comment by tchiotludo** [comment, marked as answer]
Follow the instruction here : https://akhq.io/docs/installation.html#docker
